internal/envoy/secrets.go

```diff
@@ -221,15 +221,15 @@ func (s *secretManager) watch(ctx context.Context, names []string, node string)
 		s.registry[name] = watchedCert
 	}
 	// push all newly fetched certificates into the underlying cache
-	err := s.updateCertificates(ctx, certificates)
+	err := s.updateCertificates(certificates)
 	if err != nil {
 		return err
 	}
 	return nil
 }
 
 // UnwatchAll is used to completely unwatch all a node's secrets
-func (s *secretManager) UnwatchAll(ctx context.Context, node string) error {
+func (s *secretManager) UnwatchAll(_ context.Context, node string) error {
```

Comment:
I left these here instead of removing because modifying the interface that this fulfills seemed like a bigger change than I wanted to make based on the linter today